app/javascript/packages/form-steps/use-history-param.ts

```diff
@@ -42,12 +44,12 @@ function useHistoryParam(
 
   const [value, setValue] = useState(initialValue ?? getCurrentValue);
 
-  function getValueURL(nextValue) {
+  function getValueURL(nextValue: ParamValue) {
     const prefix = typeof basePath === 'string' ? `${basePath.replace(/\/$/, '')}/` : '#';
-    return nextValue ? `${prefix}${nextValue}` : window.location.pathname + window.location.search;
+    return [prefix, nextValue].filter(Boolean).join('');
```

Comment:
I like the idea of dropping the `#` when we can! Is it worth the complexity of trying something like....?

```js
return (prefix === '#') ? nextValue : [prefix, nextValue].filter(Boolean).join('');
```

Reply:
The hash fragment feature is only currently used in selfie capture part of document capture, and I expect it will go away entirely (and the feature removed) once we absorb document capture into this flow, e.g. `/document_capture#selfie` ➡️  `/document_capture/selfie`. So I don't know that there's much motivation to improve it.

Reply:
Makes sense!